src/snowflake/connector/util_text.py

```diff
@@ -7,223 +7,86 @@
 import logging
 import re
 from io import StringIO
-from typing import Optional
 
-COMMENT_PATTERN_RE = re.compile(r"^\s*\-\-")
-EMPTY_LINE_RE = re.compile(r"^\s*$")
+from .antlr.sql_separator import sep_sqls
 
 _logger = logging.getLogger(__name__)
 
+# the pattern match a full line of comment
+sf_comment_cmd_pattern = re.compile(r"(\n|\r)+\s*--sf:.+(\n|\r)+")
```

Comment:
Module level constants are supposed to be all capital letters according to [PEP8](https://www.python.org/dev/peps/pep-0008/#constants)